### PR TITLE
authy: 2.2.2 -> 2.2.3

### PR DESCRIPTION
**Pull request by transgirllucy**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
-     * keep the list alphabetically sorted
-     * test the validity of the format with:
-         nix-build lib/tests/maintainers.nix
+  When editing this file:
+  * keep the list alphabetically sorted
+  * test the validity of the format with:
+  nix-build lib/tests/maintainers.nix
 
-    See `./scripts/check-maintainer-github-handles.sh` for an example on how to work with this data.
+  See `./scripts/check-maintainer-github-handles.sh` for an example on how to work with this data.
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:57]: Please remove those formatting changes

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -7308,10 +7308,10 @@
...
-     githubId = 55951;
-     name = "Joe DeVivo";
-   };
+    github = "joedevivo";
+    githubId = 55951;
+    name = "Joe DeVivo";
+  };
   joelancaster = {
     email = "joe.a.lancas@gmail.com";
     github = "JoeLancaster";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:7317]: We can keep those, maybe in a separate commit

File: pkgs/applications/misc/authy/default.nix [hunk 3]
```diff
@@ -10,12 +10,12 @@
...
   # curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/authy?channel=stable' | jq '.download_url,.version'
-  version = "2.2.2";
-  rev = "12";
+  version = "2.2.3";
+  rev = "18";
 
   src = fetchurl {
     url = "https://api.snapcraft.io/api/v1/snaps/download/H8ZpNgIoPyvmkgxOWw5MSzsXK1wRZiHn_${rev}.snap";
-    sha256 = "sha256-vSbqT2s5lLEU1SSDaC+sS6qt446iSKYUgKudO5WMOu4=";
+    sha256 = "b4f7dcb26e51c9551ed88b88a066eedd906848465288eedd6369983dfd1139bc";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/authy/default.nix:18]:
```suggestion
    hash = "sha256-tPfcsm5RyVUe2IuIoGbu3ZBoSEZSiO7dY2mYPf0RObw=";
```
might as well keep the sri hash

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:57]: You forgot to push this change

**Comment by mrtuxa** [comment]
i will reopen it because i struggle with git

**Pull request closed without merging**